save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/info/UserPermissions.kt

```diff
@@ -0,0 +1,15 @@
+package com.saveourtool.save.info
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property isPermittedCreateContest permission for create contests
+ * @property isPermittedToBulkUpload permission for upload COSV files
+ * @property organizationName name of organization
+ */
+@Serializable
+data class UserPermissions(
```

Comment:
my suggestion is to keep these flags on organization level, why not, @nulls 
?

Reply:
In future, those flags can be different for different users, but now they are common for all users in one organization.
I think to have them in separate object will simplify this change in future

Reply:
Changed it